lib/dgram.js

```diff
@@ -639,6 +639,25 @@ Socket.prototype.unref = function() {
   return this;
 };
 
+
+Socket.prototype.setRecvBufferSize = function(size) {
+  var err = this._handle.setRecvBufferSize(size);
+
+  if (err) {
+    throw errnoException(err, 'setRecvBufferSize');
+  }
+};
+
+
+Socket.prototype.setSendBufferSize = function(size) {
+  var err = this._handle.setSendBufferSize(size);
```

Comment:
I think we'd prefer a check in JS that will throw an `internal/errors.TypeError`
this is a good check `Number.isFinite(num) && num <= Number.MAX_SAFE_INTEGER && num >= 0;`
Ref: https://github.com/nodejs/node/blob/master/lib/internal/process.js#L31
Ref: https://github.com/nodejs/node/blob/master/lib/internal/process.js#L62